src/bindgen.rs

```diff
@@ -203,9 +203,10 @@ pub fn wasm_bindgen_build(
         "--typescript"
     };
     let target_arg = match target {
-        "nodejs" => "--nodejs",
-        "no-modules" => "--no-modules",
-        _ => "--browser",
+        Target::Nodejs => "--nodejs",
+        Target::NoModules => "--no-modules",
+        Target::Web => "--web",
+        Target::Bundler => "--browser",
```

Comment:
should we perhaps deprecate `--browser` and add a `--bundler` alias in wasm-bindgen? (you might already be on this, in which case linking to a PR/issue would be great)

Reply:
I think that long term we may want to but for now it serves a purpose we haven't displaced, so it can't be removed just yet

Reply:
do we want to open a tracking issue? let me know what you think.

Reply:
I've opened up https://github.com/rustwasm/wasm-bindgen/issues/1332 for this